### PR TITLE
ELY-2120 Avoid an NPE in ServerAuthenticationContext when the peer's …

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/ELY-2120

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@fjuma can you check this.

File: auth/server/base/src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java [hunk 1]
```diff
@@ -1057,8 +1057,12 @@ private void handleOne(final Callback[] callbacks, final int idx) throws IOExcep
                     log.tracef("Handling SocketAddressCallback");
                     if (socketAddressCallback.getKind() == SocketAddressCallback.Kind.PEER) {
                         Attributes runtimeAttributes = new MapAttributes();
+                        if ((socketAddressCallback.getAddress() != null) && ((InetSocketAddress) socketAddressCallback.getAddress()).getAddress() != null) {
                         runtimeAttributes.addFirst(KEY_SOURCE_ADDRESS, ((InetSocketAddress) socketAddressCallback.getAddress()).getAddress().getHostAddress());
                         addRuntimeAttributes(runtimeAttributes);
```
Comment (fjuma) [inline auth/server/base/src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java:1062]: These two lines should be indented now.

**Review by fjuma — CHANGES_REQUESTED** [review]
Thanks for working on this @RanabirChakraborty! Just have a small comment about formatting.

File: auth/server/base/src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java (hunk 1 above)
Comment (RanabirChakraborty) [reply auth/server/base/src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java]: Thanks :) Done

**Review by Skyllarr — APPROVED** [review]

**Pull request merged**